homeassistant/components/ialarm_xr/alarm_control_panel.py

```diff
@@ -48,7 +64,9 @@ def __init__(self, coordinator: IAlarmXRDataUpdateCoordinator) -> None:
     @property
     def state(self) -> str | None:
         """Return the state of the device."""
-        return self.coordinator.state
+        if self.coordinator.state in IALARMXR_TO_HASS:
```

Comment:
We can simplify like this:
```py
return IALARMXR_TO_HASS.get(self.coordinator.state)
```

Reply:
Pool, but I should also change the returned type of the function in Optional[str], right?

Reply:
The return value of `state` is already typed as an optional string.
```py
str | None
```
is the same as
```py
Optional[str]
```

Reply:
Uh yes, sure sorry 🙌